app/styles/package/helpers/_utils.scss

```diff
@@ -0,0 +1,54 @@
+// Default Breakpoints
+// ===================
+// These are fairly meaningless default settings.
```

Comment:
Not a huge fan of this line — I think I understand what you're getting at ("don't feel obligated to use these!"), but I think it would be good to give some reasoning about them. If I'm ever trying to justify a default breakpoint, I'll typically go with Bootstrap's: http://v4-alpha.getbootstrap.com/layout/grid/#grid-options It'd be nice to see them fleshed out a bit more like this.

It'd probably be good to break away from the idea of `desktop` and `mobile` for these too, and focus more on the size itself (kinda how Bootstrap does it).